### PR TITLE
INTYGFV-15082 & INTYGFV-15084 New layout alternatives for UeCheckboxGroup and UeRadioButtonGroup

### PR DESCRIPTION
**Pull request by SopraErik**

### CODE REVIEW

**Review by SamuelTilly — CHANGES_REQUESTED** [review]
image

image

Checkboxes gets a padding top for columns and inline layout, and when inline there is not enough space between checkboxes.

It's the same issue for radio buttons, but the extra spacing is added to the bottom instead of top for inline and columns.

Could we create a shared wrapper component that handles layout?

File: packages/common/src/types/certificate.ts [hunk 1]
```diff
@@ -158,8 +158,16 @@ export interface CheckboxCode {
   disabled?: boolean
 }
 
+export enum ConfigLayout {
+  ROWS = 'ROWS',
+  INLINE = 'INLINE',
+  COLUMN = 'COLUMN',
```
Comment (SamuelTilly) [inline packages/common/src/types/certificate.ts:164]: Doesn't look like this is used, perhaps it can be removed?

Comment (SopraErik) [reply packages/common/src/types/certificate.ts:164]: It is used in backend, so we might get error if not implemented 
I'm changing line 62 of radioGroup, which is what it's actually used for, so it becomes clearer (should be horizontal only has effect on ROWS-layout)
const shouldBeHorizontal = config.layout !== ConfigLayout.COLUMN && radiobuttons.length <= 2

Comment (SopraErik) [reply packages/common/src/types/certificate.ts:164]: INLINE-layout is meant for very shoer labels, as in drivers licence categories: C, CE, D, DE etc. and then the spacing looks good. Maybe change the storybook to reflect this

**Comment by ChrilleSigerhed** [comment]
After reviewing the implementation of Layout.COLUMN we realised it may be redundant. If we want to display an component in one column we should use Layout.ROWS. This has resulted in that COLUMN has been removed from backend. Illustration of layouts:

ROWS:
-
-
-

COLUMNS:
-  |  -
-  |  -
-  |  -

INLINE:
- - - - - - - - - -

File: packages/webcert/src/feature/certificate/Inputs/GroupWrappers.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { ConfigLayout } from '@frontend/common/'
+import styled, { css } from 'styled-components'
+
+interface checkboxesProps {
```
Comment (SamuelTilly) [inline packages/webcert/src/feature/certificate/Inputs/GroupWrappers.tsx:4]: Minor thing, but this should probably be named `GroupWrapperProps` and `checkboxProps` should be  `ItemWrapperProps`

**Review by SamuelTilly — APPROVED** [review]

**Pull request merged**